### ISSUE TITLE
Tmux error on Kali Linux 2018.4

### ISSUE THREAD

**Issue by SkypLabs** [SkypLabs/dotfiles#1]
I get this error message when opening a new GNOME terminal on Kali Linux 2018.4:

```
/home/user/.tmux.conf:44: usage: bind-key [-cnr] [-T key-table] key command [arguments]
```

**Comment by SkypLabs** [comment]
The error happens when the `if-shell` test of line 44 fails.

Two reasons seem to be the cause of the error:

- The command `bc` is not installed
- The version number contains letters (e.g. `2.9a`)

**Comment by SkypLabs** [comment]
The bug seems to still be present even after my fix.

**Comment by SkypLabs** [comment]
Fixed in a782f001.
